.github/workflows/a11y-alt-bot.yml

```
@@ -19,6 +19,7 @@ permissions:
 jobs:
   accessibility_alt_text_bot:
     name: Check alt text is set on images
+    if: ${{ ! endsWith(github.actor, '[bot]') }}
```

[Style] Super nitpicky I know 😄 but all the docs I've seen generally don't put a space after the `!`.

```suggestion
    if: ${{ !endsWith(github.actor, '[bot]') }}
```